common/logger/src/libra_logger.rs

```diff
@@ -67,6 +69,9 @@ impl LogEntry {
         let metadata = *event.metadata();
         let thread_name = thread_name.map(ToOwned::to_owned);
         let message = event.message().map(fmt::format);
+        let hostname = hostname::get()
```

Comment:
You could potentially cache this at startup since it shouldn't change, but I can't think of any problems calling this every time.

Reply:
That's a good idea, let me do that before landing